examples/package.json

```diff
@@ -32,6 +32,8 @@
     "webpack": "^1.13.1"
   },
   "dependencies": {
+    "monaco-editor": "^0.9.0",
```

Comment:
The editor is already installed since it is a dependency of the react component.